api/migrations/20210712174600-mergeIssuingAgencyNames.js

```diff
@@ -0,0 +1,99 @@
+'use strict';
+
+// list all collections that require this migration
+const collections = ['nrpti']
```

Comment:
Should include redacted_record_subset as well

Reply:
Updated `collectionAgencyPicklist` and added `redacted_record_subset` to migration

Reply:
The picklist change looks good!  However the `CollectionBCMI` schema uses `agency` field instead of `issuingAgency` so the migration script still needs to account for them

Reply:
Good catch! Should have this fixed now in the migration.